### DISCUSSION TITLE
Teacher ID rejected

### DISCUSSION THREAD

**Discussion by MushuDevs** [community/community#132608] [GitHub Education]
### Hi everyone,

Hi, I'm a teacher trying to sign up for github education, but i'm being rejected despite providing all of the associated ID etc. 
I suspect it's because the ID card and business card don't have an expiry dates on them, but that's just how our school's teacher IDs are. What else can i use to verify?
Thanks in advance.

**Comment by MushuDevs** [comment]
Bump. Any advice is welcome. I can't seem to contact anyone about this

**Reply by lindsaymarkward** [reply]
I also had this problem today. I tried it off-campus, so I'm going to try again when on campus so my IP matches, but I have no idea otherwise what the problem would be. I'm renewing after using this for years.

**Reply by MushuDevs** [reply]
Sorry to hear you're also struggling with this. It's so frustrating.

I'd be very surprised if GitHub is matching the school's IP address. How
would they know what the school's external IP is?

Anyway, please let me know how it goes 👍

**Comment by LuaanNguyen** [comment]
Hi @MushuDevs, 

I assume you also have a school email address ( .edu) or some kind of proof of employment? Maybe you could try to contact Github support for that.

**Reply by MushuDevs** [reply]
> Hi @MushuDevs, 
> 
> I assume you also have a school email address ( .edu) or some kind of proof of employment? Maybe you could try to contact Github support for that.

Hi @LuaanNguyen ,
Yep, I have my edu email verified on my account. I even got an email to the edu mailbox saying "we suspect you're a student and should apply as a student instead" in response to one of my applications which I thought was amusing. 

I of course have an employment contract but that's a far stretch from requiring an ID card, which I've already provided. 

When I tried to go through the contact form last time I was met with a "GitHub Education submissions are now automated only and we can't provide support" or something of that nature. I guess I just choose a different topic and hope they can still assist.

**Reply by LuaanNguyen** [reply]
That sucks. Well I wish you good luck and have a nice day!

**Reply by MushuDevs** [reply]
> That sucks. Well I wish you good luck and have a nice day!

Thank you
